src/irmin-pack/stats.mli

```diff
@@ -15,13 +15,15 @@
  *)
 
 module Find : sig
-  type location = Staging | Lru | Pack | Not_found [@@deriving irmin]
+  type location = Staging | Lru | Pack_direct | Pack_indexed | Not_found
```

Comment:
I wasn't immediately sure what Staging location was - I assume it is the log files?; could we add a comment just to clarify what these locations are?

Reply:
Done :+1: